### ISSUE TITLE
mekanism ores dont appear

### ISSUE THREAD

**Issue by PengyTheGamer** [Abbie5/emi-ores#22]
just trying this mod out on 1.20.1 (neo)forge and it doesn't seem like mekanism ores have any ore gen information, while other mods do

**Comment by Abbie5** [comment]
Thanks for reporting this, I can reproduce this and will look into it.

**Comment by Abbie5** [comment]
So after looking into Mekanism's code, it seems they use a [custom configured feature](https://github.com/mekanism/Mekanism/blob/94ede8cf5e3649b459b1f61566910c58a1001699/src/main/java/mekanism/common/world/ResizableOreFeatureConfig.java) for their ores. I cannot support this in my mod without depending on Mekanism's code, which I don't want to do. However, I could add an API to my mod that allows other mods to add support for their custom features... #23 

In summary, I'm afraid that showing Mekanism ores in EMI Ores is currently not possible, and might require significant work on both my and their side to change this.

**Comment by PengyTheGamer** [comment]
is it possible to add custom ore gen recipes to the tab manually?
